util/zenfs.cc

```diff
@@ -670,15 +670,24 @@ int zenfs_tool_remove_directory() {
 int zenfs_tool_restore() {
   Status status;
   IOStatus io_status;
+  IOOptions opts;
+  IODebugContext dbg;
+  bool is_dir;
 
   if (FLAGS_path.empty()) {
     fprintf(stderr, "Error: Specify --path to be restored.\n");
     return 1;
   }
 
   AddDirSeparatorAtEnd(FLAGS_restore_path);
-  AddDirSeparatorAtEnd(FLAGS_path);
-  ReadWriteLifeTimeHints();
+  std::filesystem::path fpath(FLAGS_path);
+  FLAGS_path = fpath.lexically_normal().string();
+  FileSystem *f_fs = FileSystem::Default().get();
+  status = f_fs->IsDirectory(FLAGS_path, opts, &is_dir, &dbg);
+  if (!status.ok()) {
+    fprintf(stderr, "Failed with %s\n", status.ToString().c_str());
```

Comment:
Please print what failed,align with the other IsDirectory error print

Reply:
This does print what failed. Here is the error print in case of failure here:
"Failed with IO error: While open for IsDirectory(): /tmp/bkp1/chennai/: Not a directory "

Anyhow, I have aligned this to the other error print as below:
fprintf(stderr, "IsDirectory failed, error: %s\n",<snip>);